README.md

```diff
@@ -82,7 +77,8 @@ npm run electron
 
 ## Mobile app
 
-Using [Cordova](https://cordova.apache.org/), you can run the hoverpad as an application on your mobile.
+Using [Cordova](https://cordova.apache.org/), you can run the hoverpad
+as an application on your mobile.
```

Comment:
"Using Cordova allows you to create a  Hoverpad binary which can be installed on your mobile Android/iOS device."?